src/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -222,17 +224,40 @@ export const SavedSearchAlertModal: FC<SavedSearchAlertFormProps> = ({
               <SavedSearchAlertNameInputQueryRenderer />
               <Box>
                 <Text variant="xs">Filters</Text>
-                <Spacer y={2} />
+                <Spacer y={1} />
                 <Flex flexWrap="wrap" gap={1}>
                   <SavedSearchAlertPills
                     items={pills}
                     onDeletePress={handleRemovePillPress}
                   />
                 </Flex>
 
-                <Separator my={1} />
+                <Separator my={2} />
+
+                <Media at="xs">
+                  <PriceRangeFilter expanded={false} />
+                </Media>
+                <Media greaterThan="xs">
+                  <PriceRangeFilter expanded={true} />
+                </Media>
+
+                <Separator my={2} />
 
-                <PriceRangeFilter />
+                <Text variant="sm-display">
+                  Tell us more about what you’re looking for
+                </Text>
+                <Spacer y={1} />
+                <TextArea
+                  name="details"
+                  placeholder="For example, a specific request such as ‘spin paintings series’ or ‘signed prints’"
```

Comment:
**suggestion**: What do you think of moving these strings to our translations file? We already have a section for this modal [here](https://github.com/artsy/force/blob/main/src/System/i18n/locales/en-US/translation.json#L279).

Reply:
I also updated [the Jira ticket](https://artsyproduct.atlassian.net/browse/ONYX-433?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ) with recently revised copy.

Reply:
Yea - I'm definitely opinionated that the internationalization stuff currently is just kind of a pain from a DX perspective, and I've never/am doubtful that we'll see any hypothetical 'wins' from it.

This is probably not the PR to discuss it, but requiring its use worsens DX in a couple of real ways:
  - divorces copy from its location in the code. this makes it harder to quickly find/debug things (can't grep for copy), and is different than the co-location we typically prefer for most things
  - adds another layer of abstraction/naming - a dev needs to figure out some sensible naming + structure in a separate (currently monolithic) file

The benefits to maintaining the separate copy file, which I've not yet actually seen ever happen in production:
  - actual translations and offering content in other languages more seamlessly - this would likely be a large undertaking and i don't see this on any roadmap
  - allowing non-devs to more easily audit and update copy - again, i've simply not seen this ever happen in practice. would PM's or designers (or copy people) be doing this? i don't think this is likely to happen.

So for me, I would decide not to be pursuing this in Force.

Reply:
feel free to chime in @dzucconi if you've had other thoughts.

Reply:
My main beef was with i18n in Volt and other backend apps, where the possibility of localization seemed close to zero. 

For Force, it's at least _plausible_, though it of course hasn't happened yet.

I agree about the DX friction — tried to improve it a tiny bit by adding [the `yarn translations` command](https://github.com/artsy/force/pull/12038), but that can only do so much.

Reply:
I am not strongly opinionated, but I think we should agree to either use it or not in Force. The same discussion happened a few months ago: https://github.com/artsy/force/pull/12693#discussion_r1283209475

@tam-kis should we discuss it on web practice next time?

Reply:
For now, feel free to hardcore the copy and skip the translation file.